# ДЗ №5

## Задание 1
Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. 

    Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

$T_{1,2} = \overline{X} \pm Z_{\frac{a}{2}} \cdot \frac{\sigma}{\sqrt{n}}$

$T_{1,2} = \overline{X} \pm Z_{\frac{0.05}{2}} \cdot \frac{16}{\sqrt{256}}$

$T_{1,2} = \overline{X} \pm 1.96$

$T_1 = 80 + 1.96 = 81.96$

$T_2 = 80 - 1.96 = 78.04$

## Задание 2
В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 
    
    Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

In [2]:
values = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]

$T_{1,2} = \overline{X} \pm t_{\frac{a}{2}} \cdot \frac{\sigma}{\sqrt{n}}$

$a = 0.05$

$v = 9$

$t = 2.262$

In [17]:
import numpy as np
std = np.std(values, ddof=1)
std

0.4508017549014448

In [18]:
np.mean(values)

6.590000000000001

In [19]:
T1 = 6.59 + 2.262*(std/10**0.5)
T1

6.912461744087574

In [20]:
T2 = 6.59 - 2.262*(std/pow(10, 0.5))
T2

6.2675382559124255

## Задание 3
Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.

    Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

$$Z_н = \frac{\overline X * M_0}{\frac{\sigma}{\sqrt{n}}}$$

$Z_н = \frac{17.5 - 17}{\frac{2}{\sqrt{100}}}$

In [22]:
Zn = 0.5/(2/100**0.5)
Zn

2.5

In [23]:
Z = 1.645

$Z_n > Z => H_1 $ верна

## Задание 4
Продавец утверждает, что средний вес пачки печенья составляет 200 г.
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.

    Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?
    
H0 - средний вес пачки - 200гр

H1 - средний вес пачки не 200гр

---

$a = 0.01$

$Z = \pm 3.25$

In [26]:
values1 = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]

In [27]:
np.mean(values1)

198.5

In [29]:
std1 = np.std(values1, ddof=1)
std1

4.453463071962462

$$Z_н = \frac{\overline X * M_0}{\frac{\sigma}{\sqrt{n}}}$$

$Z_н = \frac{198.5 - 200}{\frac{4.453}{\sqrt{10}}}$

In [32]:
Zn = (198.5-200)/(std1/10**0.5)
Zn

-1.0651074037450896

Zn - попадает в зону H0

Поэтому принимаем нулевую гипотезу